app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    result = {}
+
+    def inner(*args) -> Callable:
+
+        if args in result:
```

Comment:
and what if I use func like that
`long_time_func(a=1, b=2, c=3)`
I will receive an error
`TypeError: cache.<locals>.inner() got an unexpected keyword argument 'a'`
update your code by adding **kwargs